### ISSUE TITLE
3D View - Default tool mode should be "Active Tool". 

### ISSUE THREAD

**Issue by Draise14** [Bforartists/Bforartists#3073]
![image](https://user-images.githubusercontent.com/35357409/174620175-6c9804e0-eb6f-4f94-9038-4d958cd353b8.png)

For some reason, a default startup has all tool default to "Tweak" instead of "Active Tool". meaning.. I get the same viewport manipulation behavior regardless of changing the tool. I have had some beginning interns and students voice their frustration (while watching them live), with a couple others in Discord mentions something - and I realized with the factory default I too would be frustrated. It's not how I have my defaults, and changing transform modes should have predictable behaviour and expected UX. 

**The UX should be this:**
Press W - Goes to transform - use widget or viewport to transform
Press E - Goes to rotation - use widget or viewport to rotate
Press R - Goes to scale - use widget or viewport to scale. 

**Current behaviour:**
Press W - Goes to trasnform - use widget - anywhere in viewport tweaks in transform, ok
Press E - Goes to rotation - use widget - anywhere in viewport tweaks in transform, what?
Press R - Goes to scale - use widget - anywhere in viewport tweaks in transform, huh?

**Task**
- Find how these defaults are stored
- Set to "Active Tool" where possible

**Comment by ReinerBforartists** [comment]
Hm. Active tool means we could end in box select all the time, like in Blender ...

**Comment by Draise14** [comment]
This doesn't happen in the Bforartists keymap.

Here is the working and expected behaviour:
![bforartists_9urBpTb739](https://user-images.githubusercontent.com/35357409/174640131-bdf23e0e-893b-47ce-898b-1573a69369ff.gif)
 
 Here is the current default behaviour, and broken UX. 
![bforartists_5y1GYdOLIN](https://user-images.githubusercontent.com/35357409/174640298-50adbc89-4ec3-4ddb-a9ad-1e2b707e8cc4.gif)

**Comment by ReinerBforartists** [comment]
I see. So i guess we need to resave all startup.blends again ...

I seem to remember that i toyed around with these tool settings before. But i can't remember where and when. Must have happened shortly after they got introduced.

**Comment by ReinerBforartists** [comment]
We had this fixed already. Related: 

https://github.com/Bforartists/Bforartists/issues/1443

**Comment by Draise14** [comment]
Yeah, I think this is due to keymap handling of tweak, press and click that has improved this year in the Blender source.

**Comment by ReinerBforartists** [comment]
Let's fix this ...

**Comment by ReinerBforartists** [comment]
Fixed in master

**Comment by Draise14** [comment]
This one got regressed, so should be implemented again. Argument is sound still.

**Comment by ReinerBforartists** [comment]
It would have been better to make a new issue since this issue here is old, was fixed and closed. Just as a hint. It also contained my name as the assignee then. Which i am of course not anymore.

**Comment by Draise14** [comment]
Ah sure, will make a new one tomorrow

**Comment by Blenux** [comment]
Think this one can be closed, current codebase defaults to Active Tool.

**Comment by Draise14** [comment]
Was this tested in all workspace? Still gotta do that before making a new task.

**Comment by Draise14** [comment]
Tested, yeah.. fixed itself.
